Jenkinsfile.jssdk

```diff
@@ -0,0 +1,44 @@
+#!groovy
+ansiColor('xterm') {
+  timestamps {
+    timeout(60) {
+      node('NODE_JS_BUILDER') {
+        stage('Install') {
+          withCredentials([
+            string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
+          ]) {
+            sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' > $HOME/.npmrc'
+            sh '''#!/bin/bash -ex
+            source ~/.nvm/nvm.sh
+            nvm install 7
+            nvm use v7
+            npm install
+            rm -f $HOME/.npmrc
+            '''
+          }
+        }
+
+        stage('Upgrade') {
+          echo "Upgrading @ciscospark packages"
+          sh '''#!/bin/bash -ex
+          source ~/.nvm/nvm.sh
+          nvm use v7
+          npm run upgradespark
+          '''
+        }
+
+        stage('Branching') {
+          echo "Creating Branch"
+          sh '''#!/bin/bash -ex
+          version=`npm view ciscospark version`
+          git add package.json
+          git commit -m "chore: update @ciscospark packages to ${version}"
+          '''
+          sshagent(['6c8a75fb-5e5f-4803-9b6d-1933a3111a34']) {
+            sh "git push upstream HEAD:js-sdk/${version}"
```

Comment:
nope. the version variable is only available in the original sh block. 

you could store it with a line like

```groovy
def version = sh script: 'npm view ciscospark version', returnStdout: true
version = version.trim()
```